mapie/estimator/regressor.py

```diff
@@ -497,7 +497,8 @@ def predict(
         self,
         X: ArrayLike,
         ensemble: bool = False,
-        return_multi_pred: bool = True
+        return_multi_pred: bool = True,
+        **predict_params
```

Comment:
We change the signature of the `predict` method in the abstract class `EnsembleEstimator` because the inherit classes `EnsembleClassifier` and `EnsembleRegressor` don't need the same parameters. To avoid conflict in the parameters types, we conserve the `**kwargs`, named `**predict_params` here.